### DISCUSSION TITLE
ctrl+d to select word at cursor

### DISCUSSION THREAD

**Discussion by GaZaTu** [Chatterino/chatterino2#3628] [Ideas]
title

edit: btw to clarify: i don't mean mouse cursor, i mean text cursor inside the input

**Comment by Mm2PL** [comment]
Easy enough to do as an action. But there needs to be a good definition of a word. `(?:^|\s)(\w+)(?:\s|$)`? How about words with - in the middle.

**Reply by GaZaTu** [reply]
same as double click, which i assume is just `\w+` (so no dash)

**Comment by pajlada** [comment]
Fixed in 29117a930e8a4c294f6ad2fd35e115cc992742e0
